incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -468,6 +469,7 @@ public MessageConsumer newStream(
             final long budgetId = supplyBudgetId.getAsLong();
 
             newStream = new MqttServer(
+                binding,
```

Comment:
We don't typically pass the binding into the stream implementation, instead we keep these separate and extract what we need from the binding then pass to the constructor.